packages/p2p/src/components/my-ads/create-ad-form.jsx

```diff
@@ -95,6 +95,17 @@ const CreateAdForm = () => {
                 }}
             >
                 {({ errors, handleChange, isSubmitting, isValid, setFieldValue, touched, values }) => {
+                    if (isNaN(values.offer_amount)) {
```

Comment:
It is best if we use: `declarative-validation-rules` module for this to avoid additional logic. You can check `validAmount` and pass in additional options as well as `required` rule.